Source/Urho3D/LuaScript/LuaScript.h

```diff
@@ -71,6 +71,12 @@ class URHO3D_API LuaScript : public Object, public LuaScriptEventListener
     bool ExecuteFile(const String& fileName);
     /// Execute script string. Return true if successful.
     bool ExecuteString(const String& string);
+    /// Load script file on file system (i.e. not from resource cache).

```

Comment:
Based on our coding convention, the comment on headers should be a one-liner to keep Doxygen happy. If the comment is really long (which is not in this particular case) then you could use the ```/// \brief``` at the beginning of the comment.